.github/workflows/test.yml

```diff
@@ -59,47 +60,60 @@ jobs:
       PYTHONDEVMODE: '1'
     steps:
       - name: Set up Python
-        uses: actions/setup-python@v5
+        uses: Quansight-Labs/setup-python@v5
         with:
           python-version: ${{ matrix.py }}
           architecture: ${{ matrix.arch }}
 
       - name: Install Rust
-        if: matrix.arch == 'x64'
+        if: matrix.arch == 'x64' && !startsWith(matrix.py, '3.13')
         uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: stable
 
       - uses: actions/checkout@v4
 
       - name: Install Dependencies
+        if: "!endsWith(matrix.py, 't')"
         shell: bash
         run: |
           python -m pip install --require-hashes -r ci/requirements.txt
 
+      - name: Install Dependencies (free-threading)
+        if: "endsWith(matrix.py, 't')"
+        shell: bash
+        run: |
+          python -m pip install --require-hashes -r ci/requirements.freethreading.txt
+
       # TODO enable once PyO3 supports 3.13.
       - name: Build (Rust)
-        if: matrix.arch == 'x64' && matrix.py != '3.13'
+        if: matrix.arch == 'x64' && !startsWith(matrix.py, '3.13')
```

Comment:
I'm guessing this is supported now. But scope bloat to resolve it in this PR.

Reply:
I opened a followup issue to track this https://github.com/indygreg/python-zstandard/issues/251